extensions/amp-bind/0.1/test/test-bind-evaluator.js

```diff
@@ -66,35 +70,49 @@ describe('BindEvaluator', () => {
       property: 'text',
       expressionString: 'oneplusone + 2',
     };
-    expect(evaluator.bindingsForTesting().length).to.equal(0);
+    expect(numberOfBindings()).to.equal(0);
     evaluator.addBindings([{
       tagName: 'P',
       property: 'text',
       expressionString: 'oneplusone + 2',
     }]);
-    expect(evaluator.bindingsForTesting().length).to.equal(1);
-    const results = evaluator.evaluateBindings({oneplusone: 2});
-    const evaluated = results['results'];
-    const errors = results['errors'];
+    expect(numberOfBindings()).to.equal(1);
+    const {results, errors} = evaluator.evaluateBindings({oneplusone: 2});
+    expect(results['oneplusone + 2']).to.equal(4);
     expect(errors['oneplusone + 2']).to.be.undefined;
-    expect(evaluated['oneplusone + 2']).to.not.be.undefined;
-    expect(evaluated['oneplusone + 2'] = '4');
   });
 
   it('should treat out-of-scope vars as null', () => {
-    expect(evaluator.bindingsForTesting().length).to.equal(0);
+    expect(numberOfBindings()).to.equal(0);
     evaluator.addBindings([{
       tagName: 'P',
       property: 'text',
       expressionString: 'outOfScope',
     }]);
-    expect(evaluator.bindingsForTesting().length).to.equal(1);
-    const results = evaluator.evaluateBindings({});
-    const evaluated = results['results'];
-    const errors = results['errors'];
+    expect(numberOfBindings()).to.equal(1);
+    const {results, errors} = evaluator.evaluateBindings({});
+    expect(results['outOfScope']).to.be.null;
     expect(errors['outOfScope']).to.be.undefined;
-    expect(evaluated['outOfScope']).to.not.be.undefined;
-    expect(evaluated['outOfScope']).to.be.null;
   });
 
+  it('should validate a common expression on each respective binding', () => {
+    const string = /* eslint no-script-url: 0 */ '"javascript:alert(1)"';
+    evaluator.addBindings([{
+      tagName: 'P',
+      property: 'text',
+      expressionString: string,
+    }]);
+    let {results, errors} = evaluator.evaluateBindings({});
+    expect(results[string])
```

Comment:
How about asserting that there's an error as well, just for parallelism with the below checks?